Jadudm/remove migration db (#3063)
GSA-TTS/FAC: backend/census_historical_migration/README.md

1.  Download test Census data from https://drive.google.com/drive/folders/1TY-7yWsMd8DsVEXvwrEe_oWW1iR2sGoy into census_historical_migration/data folder.
NOTE:  Never check in the census_historical_migration/data folder into GitHub.

> 2.  In the FAC/backend folder, run the following to load CSV files from census_historical_migration/data folder into fac-census-data-s3 bucket.

Reviewer: ```suggestion
2.  In the FAC/backend folder, run the following to load CSV files from census_historical_migration/data folder into gsa-fac-private-s3 bucket.
```

```bash
docker compose run --rm web python manage.py fac_s3 \

>   fac-census-data-s3 \

Reviewer: ```suggestion
  gsa-fac-private-s3 \
```

  --upload \
  --src census_historical_migration/data
```

> 3.  In the FAC/backend folder, run the following to read the CSV files from fac-census-data-s3 bucket and load into Postgres.

Reviewer: ```suggestion
3.  In the FAC/backend folder, run the following to read the CSV files from gsa-fac-private-s3 bucket and load into Postgres.
```
